src/memory/AmdFlash.cc

```diff
@@ -8,6 +8,7 @@
 #include "MSXCliComm.hh"
 #include "MSXException.hh"
 
+#include "enumerate.hh"
```

Comment:
Maybe I overlooked it, but this patch doesn't use `enumerate`. Is this a left over from some experiment?